util/CMakeLists.txt

```diff
@@ -27,6 +27,9 @@ if (eigen3_FOUND)
     LIBS -ldrakeUtil
     REQUIRES
     VERSION 0.0.1)
+  add_library(drakeContactConstraintsUtil SHARED drakeContactConstraintsUtil.cpp)
```

Comment:
No problem.  We discussed moving the helper methods into their own file, but not the specifics of where it should go.  Should be quick work to move this out of Utils.